spec/browsers/webkit_user_agent_spec.rb

```diff
@@ -1241,3 +1241,35 @@
 
   it { expect(@useragent).to be_mobile }
 end
+
+describe 'Mozilla/5.0 (iPhone; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) FxiOS/1.0 Mobile/12F69 Safari/600.1.4' do
+  before do
+    @useragent = UserAgent.parse('Mozilla/5.0 (iPhone; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) FxiOS/1.0 Mobile/12F69 Safari/600.1.4')
+  end
+
+  it "should return 'Firefox' as its browser" do
+    expect(@useragent.browser).to eq('Firefox')
+  end
+
+  it "should return '600.1.4' as its build" do
+    expect(@useragent.build).to eq('600.1.4')
+  end
+
+  it "should return nil as its version" do
+    expect(@useragent.version).to be_nil
```

Comment:
I think you should probably make a new browser class which properly supports this; I would expect `1.0` to be returned for this UA